app/views/users/show.html.haml

```diff
@@ -0,0 +1,48 @@
+.container-fluid
+  .container.users_exterior_container
+    .row.users_container
+      .col.d-flex.new-post-container
+        = link_to "New Post", new_post_path, class: 'new_post_button'
+      .container.show_user_exterior_container.justify-content-start
+        .col-md-12
+          .container.users_back_container
+            = link_to "Back", root_path, class: 'post_back_button'
+          .container.show_user_container.justify-content-start
+            .container.user-information.d-flex
+              - if @user.profile_picture.attached?
+                = image_tag url_for(@user.profile_picture), class: 'user-picture', alt: 'User Image'
+              - else
+                = image_tag 'user_picture.jpg', class: 'user-picture', alt: 'Default Profile Image'
+              .container
+                .container.d-flex
+                  %p.user_name #{@user.first_name} #{@user.last_name}
+                  %p.user_nickname @#{@user.nickname}
+                .container.d-flex.user_dates
+                  %p.user_birthday Born #{@user.birthday.strftime("%m/%d/%Y")}
+                  %p Joined #{@user.created_at.strftime("%m/%d/%Y")}
+            %hr.users_profile_separation
+              .container.user-stats-content
+                %p.post-count #{@user.posts.count}
+                %p Posts
+            %hr.users_profile_separation
+            .container
+              .col-md-12
+                - @user.posts.each_with_index do |post, index|
+                  .profile-post-container
+                    .row
+                      .col-md-2
+                        - if @user.profile_picture.attached?
+                          = image_tag url_for(@user.profile_picture), alt: 'User Image', class: 'post-user-image'
+                        - else
+                          = image_tag 'user_picture.jpg', alt: 'Default Profile Image', class: 'post-user-image'
+                      .col-md-9
+                        .post-container
+                          %p.fullname_paragraph #{@user.first_name} #{@user.last_name}
+                          %p @#{@user.nickname}
+                          .container.post-content
+                            %h5= post.title
+                            %p= post.body
+                            - if post.image.attached?
+                              = image_tag url_for(post.image), alt: 'Post Image', class: 'post-image'
+                    - unless index == @user.posts.size - 1
+                      %hr.profile_post_separation
```

Comment:
I think this logic is really similar to the one used in the index posts.

You could define a partial, and use it in both the posts index and the user show